docs/CONTRIBUTING.md

```diff
@@ -224,13 +224,14 @@ following manner:
   the `reviewers wanted` label.  If for some reason, no reviewers have
   been assigned after three days, the author of the pull request
   should feel free to ping `@sxs-collaboration/spectre-core-devs`
-- Anyone can request changes within the first two days.  After two
-  days only the assigned reviewers can request changes, unless someone
-  believes the code is wrong.  Non-reviewers are allowed to make
-  comments, which the pull request author is encouraged to address.
-  Alternatively the author can create an issue with the suggested
-  changes, assigned to themselves, which would then be addressed in a
-  subsequent pull request by the author.
+- Anyone can request changes within the first two days (or within a
+  day after two approvals on the first day).  After two days only the
```

Comment:
`max(2 days, [assigned reviews submitted] + 1 day)`, not `min(2 days, [assigned reviews submitted] + 1 day)`